Mergin/version_viewer_dialog.py

```diff
@@ -0,0 +1,588 @@
+from collections import deque
+import os
+
+from qgis.PyQt import uic, QtCore
+from qgis.PyQt.QtWidgets import QDialog, QAction, QListWidgetItem, QPushButton, QMenu, QMessageBox
+from qgis.PyQt.QtGui import QStandardItem, QStandardItemModel, QIcon, QFont, QColor
+from qgis.PyQt.QtCore import (
+    QStringListModel,
+    Qt,
+    QSettings,
+    QModelIndex,
+    QAbstractTableModel,
+    QThread,
+    pyqtSignal,
+    QItemSelectionModel,
+)
+
+from qgis.utils import iface
+from qgis.core import QgsProject, QgsMessageLog, QgsApplication, QgsFeatureRequest, QgsVectorLayerCache
+from qgis.gui import QgsGui, QgsMapToolPan, QgsAttributeTableModel, QgsAttributeTableFilterModel
+
+
+from .utils import (
+    ServerType,
+    ClientError,
+    LoginError,
+    InvalidProject,
+    check_mergin_subdirs,
+    create_mergin_client,
+    find_qgis_files,
+    get_mergin_auth,
+    icon_path,
+    mm_symbol_path,
+    is_number,
+    login_error_message,
+    mergin_project_local_path,
+    PROJS_PER_PAGE,
+    remove_project_variables,
+    same_dir,
+    unhandled_exception_message,
+    unsaved_project_check,
+    UnsavedChangesStrategy,
+    contextual_date,
+    is_versioned_file,
+    icon_path,
+    format_datetime,
+    parse_user_agent
+)
```

Comment:
Quite a few of these imports are not used in this file

Reply:
Removed those in last version